pycbc/events/triggers.py

```diff
@@ -283,6 +283,8 @@ def get_param(par, args, m1, m2, s1z, s2z):
     elif par in ['chi_eff', 'effective_spin']:
         parvals = conversions.chi_eff(m1, m2, s1z, s2z)
     elif par == 'template_duration':
+        if not hasattr(args, 'approximant'):
+            args.approximant = None
```

Comment:
should this not be defaulted to "SEOBNRv4" - as the next line ignores any None values anyway?

Reply:
I don't want to have 'SEOBNRv4' hard-coded as default in two places. Can reorganize a bit to achieve that.